libs/dh/auth/feature-msal/src/lib/b2c-config.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { MsalInterceptorConfiguration } from '@azure/msal-angular';
+import {
+  BrowserCacheLocation,
+  InteractionType,
+  IPublicClientApplication,
+  LogLevel,
+  PublicClientApplication,
+} from '@azure/msal-browser';
+
+import { MsalGuardConfiguration } from './@azure/msal-angular/msal.guard.config';
+
+/**
+ * Enter here the coordinates of your web API and scopes for access token request
+ * The current application coordinates were pre-registered in a B2C tenant.
+ * To learn more about user flows, visit: https://docs.microsoft.com/en-us/azure/active-directory-b2c/user-flow-overview
+ * To learn more about custom policies, visit: https://docs.microsoft.com/en-us/azure/active-directory-b2c/custom-policy-overview
+ */
+export const apiConfig: { scopes: string[]; uri: string } = {
+  scopes: ['https://fabrikamb2c.onmicrosoft.com/helloapi/demo.read'],
+  uri: 'https://fabrikamb2chello.azurewebsites.net/hello',
+};
+
+export function MSALInstanceFactory(): IPublicClientApplication {
+  return new PublicClientApplication({
+    auth: {
+      clientId: '88e5d356-0c71-49e9-b260-d0629f3c0445',
+      authority:
+        'https://dev002DataHubB2C.b2clogin.com/dev002DataHubB2C.onmicrosoft.com/B2C_1_sign_in_experiments',
```

Comment:
question: Are all of the hardcoded url values in this file the same across environments? If they aren't, we'll have an issue as soon as we deploy to another env. We'll have to think of a strategy for that when it becomes an issue.

Reply:
Right now we only have a single environment for B2C, not sure if multiple environments are going to be created in future. So yes these value is hardcoded for all environments for now.